<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1020</width>
    <height>720</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>960</width>
    <height>720</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1920</width>
    <height>1080</height>
   </size>
  </property>
  <property name="palette">
   <palette>
    <active>
     <colorrole role="WindowText">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Button">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>102</green>
        <blue>25</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Light">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>199</green>
        <blue>170</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Midlight">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>144</green>
        <blue>89</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Dark">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>127</red>
        <green>51</green>
        <blue>12</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Mid">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>170</red>
        <green>68</green>
        <blue>16</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Text">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>0</red>
        <green>0</green>
        <blue>0</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="BrightText">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="ButtonText">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>0</red>
        <green>0</green>
        <blue>0</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Base">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Window">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>102</green>
        <blue>25</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Shadow">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>0</red>
        <green>0</green>
        <blue>0</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="AlternateBase">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>178</green>
        <blue>140</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="ToolTipBase">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>220</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="ToolTipText">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>0</red>
        <green>0</green>
        <blue>0</blue>
       </color>
      </brush>
     </colorrole>
    </active>
    <inactive>
     <colorrole role="WindowText">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Button">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>102</green>
        <blue>25</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Light">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>199</green>
        <blue>170</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Midlight">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>144</green>
        <blue>89</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Dark">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>127</red>
        <green>51</green>
        <blue>12</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Mid">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>170</red>
        <green>68</green>
        <blue>16</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Text">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>0</red>
        <green>0</green>
        <blue>0</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="BrightText">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="ButtonText">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>0</red>
        <green>0</green>
        <blue>0</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Base">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Window">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>102</green>
        <blue>25</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Shadow">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>0</red>
        <green>0</green>
        <blue>0</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="AlternateBase">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>178</green>
        <blue>140</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="ToolTipBase">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>220</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="ToolTipText">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>0</red>
        <green>0</green>
        <blue>0</blue>
       </color>
      </brush>
     </colorrole>
    </inactive>
    <disabled>
     <colorrole role="WindowText">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>127</red>
        <green>51</green>
        <blue>12</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Button">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>102</green>
        <blue>25</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Light">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>199</green>
        <blue>170</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Midlight">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>144</green>
        <blue>89</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Dark">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>127</red>
        <green>51</green>
        <blue>12</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Mid">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>170</red>
        <green>68</green>
        <blue>16</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Text">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>127</red>
        <green>51</green>
        <blue>12</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="BrightText">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="ButtonText">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>127</red>
        <green>51</green>
        <blue>12</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Base">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>102</green>
        <blue>25</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Window">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>102</green>
        <blue>25</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Shadow">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>0</red>
        <green>0</green>
        <blue>0</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="AlternateBase">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>102</green>
        <blue>25</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="ToolTipBase">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>220</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="ToolTipText">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>0</red>
        <green>0</green>
        <blue>0</blue>
       </color>
      </brush>
     </colorrole>
    </disabled>
   </palette>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1020</width>
     <height>20</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuARCHIVO">
    <property name="title">
     <string>ARCHIVO</string>
    </property>
    <addaction name="actionCONFIGURACI_N_DE_OPCIONES"/>
    <addaction name="actionUsuarios_2"/>
   </widget>
   <widget class="QMenu" name="menuACTUALIZACIONES">
    <property name="title">
     <string>ACTUALIZACIONES</string>
    </property>
    <addaction name="actionClientes"/>
    <addaction name="actionCobradores"/>
   </widget>
   <widget class="QMenu" name="menuOPERACIONES">
    <property name="title">
     <string>OPERACIONES</string>
    </property>
    <addaction name="actionAGREGAR_NUEVOS_CR_DITOS"/>
    <addaction name="actionGENERADOR_CA_DA_DE_PR_STAMOS"/>
    <addaction name="actionREGLAS_OTORGACI_N_DE_CR_DITOS"/>
    <addaction name="actionREGLAS_EMISI_N_DE_CR_DITOS"/>
    <addaction name="actionREPORTE_DE_CR_DITOS"/>
    <addaction name="actionHISTORIAL_CR_DITOS_Y_CUOTAS"/>
    <addaction name="actionBAJA_CR_DITOS"/>
    <addaction name="actionMODIFICACI_N_DE_CR_DITOS"/>
    <addaction name="actionREPORTE_DE_CUOTAS"/>
    <addaction name="actionCOBRAR_CUOTAS"/>
    <addaction name="actionPUNITORIOS_DE_CUOTAS"/>
    <addaction name="actionGENERADOR_DE_COMISIONES"/>
    <addaction name="actionREFINANCIAMIENTO"/>
    <addaction name="actionCONSULTA_DEUDORES"/>
    <addaction name="actionREPORTE_DEUDORES"/>
    <addaction name="actionBUSCAR_CLIENTES"/>
   </widget>
   <widget class="QMenu" name="menuESTADISTICAS">
    <property name="title">
     <string>ESTADISTICAS</string>
    </property>
    <addaction name="actionGR_FICOS"/>
   </widget>
   <widget class="QMenu" name="menuAYUDA">
    <property name="title">
     <string>AYUDA</string>
    </property>
    <addaction name="actionCONTENIDOS"/>
    <addaction name="actionSITIOS_WEB"/>
   </widget>
   <addaction name="menuARCHIVO"/>
   <addaction name="menuACTUALIZACIONES"/>
   <addaction name="menuOPERACIONES"/>
   <addaction name="menuESTADISTICAS"/>
   <addaction name="menuAYUDA"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QToolBar" name="mainToolBar">
   <property name="iconSize">
    <size>
     <width>50</width>
     <height>50</height>
    </size>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionCONFIGURAR_OPCIONES"/>
   <addaction name="actionCOBRADORES"/>
   <addaction name="actionUsuarios"/>
   <addaction name="actionCLIENTES"/>
  </widget>
  <action name="actionCONFIGURAR_OPCIONES">
   <property name="icon">
    <iconset>
     <normaloff>../Escritorio/Iconos/settings.png</normaloff>
     <normalon>../Escritorio/Iconos/settings.png</normalon>../Escritorio/Iconos/settings.png</iconset>
   </property>
   <property name="text">
    <string>CONFIGURAR OPCIONES</string>
   </property>
  </action>
  <action name="actionCLIENTES">
   <property name="icon">
    <iconset>
     <normaloff>../Escritorio/Iconos/profile_avatar_placeholder_large.png</normaloff>../Escritorio/Iconos/profile_avatar_placeholder_large.png</iconset>
   </property>
   <property name="text">
    <string>CLIENTES</string>
   </property>
  </action>
  <action name="actionCOBRADORES">
   <property name="icon">
    <iconset>
     <normaloff>../Escritorio/Iconos/profile_avatar_ninja2.png</normaloff>../Escritorio/Iconos/profile_avatar_ninja2.png</iconset>
   </property>
   <property name="text">
    <string>Cobradores</string>
   </property>
  </action>
  <action name="actionAGREGAR_NUEVOS_CR_DITOS">
   <property name="text">
    <string>AGREGAR NUEVOS CRÉDITOS</string>
   </property>
  </action>
  <action name="actionGENERADOR_CA_DA_DE_PR_STAMOS">
   <property name="text">
    <string>GENERADOR CAÍDA DE PRÉSTAMOS</string>
   </property>
  </action>
  <action name="actionREGLAS_OTORGACI_N_DE_CR_DITOS">
   <property name="text">
    <string>REGLAS OTORGACIÓN DE CRÉDITOS</string>
   </property>
  </action>
  <action name="actionREGLAS_EMISI_N_DE_CR_DITOS">
   <property name="text">
    <string>REGLAS EMISIÓN DE CRÉDITOS</string>
   </property>
  </action>
  <action name="actionREPORTE_DE_CR_DITOS">
   <property name="text">
    <string>REPORTE DE CRÉDITOS</string>
   </property>
  </action>
  <action name="actionHISTORIAL_CR_DITOS_Y_CUOTAS">
   <property name="text">
    <string>HISTORIAL CRÉDITOS Y CUOTAS</string>
   </property>
  </action>
  <action name="actionBAJA_CR_DITOS">
   <property name="text">
    <string>BAJA CRÉDITOS</string>
   </property>
  </action>
  <action name="actionMODIFICACI_N_DE_CR_DITOS">
   <property name="text">
    <string>MODIFICACIÓN DE CRÉDITOS</string>
   </property>
  </action>
  <action name="actionREPORTE_DE_CUOTAS">
   <property name="text">
    <string>REPORTE DE CUOTAS</string>
   </property>
  </action>
  <action name="actionCOBRAR_CUOTAS">
   <property name="text">
    <string>COBRAR CUOTAS</string>
   </property>
  </action>
  <action name="actionPUNITORIOS_DE_CUOTAS">
   <property name="text">
    <string>PUNITORIOS</string>
   </property>
  </action>
  <action name="actionGENERADOR_DE_COMISIONES">
   <property name="text">
    <string>GENERADOR DE COMISIONES</string>
   </property>
  </action>
  <action name="actionREFINANCIAMIENTO">
   <property name="text">
    <string>REFINANCIAMIENTO</string>
   </property>
  </action>
  <action name="actionCONSULTA_DEUDORES">
   <property name="text">
    <string>CONSULTA DEUDORES</string>
   </property>
  </action>
  <action name="actionREPORTE_DEUDORES">
   <property name="text">
    <string>REPORTE DEUDORES</string>
   </property>
  </action>
  <action name="actionGR_FICOS">
   <property name="text">
    <string>GRÁFICOS</string>
   </property>
  </action>
  <action name="actionCONTENIDOS">
   <property name="text">
    <string>CONTENIDOS</string>
   </property>
  </action>
  <action name="actionSITIOS_WEB">
   <property name="text">
    <string>SITIOS WEB</string>
   </property>
  </action>
  <action name="actionCONFIGURACI_N_DE_OPCIONES">
   <property name="icon">
    <iconset>
     <normalon>../Escritorio/Iconos/settings.png</normalon>
    </iconset>
   </property>
   <property name="text">
    <string>Configuración de Opciones</string>
   </property>
  </action>
  <action name="actionBUSCAR_CLIENTES">
   <property name="text">
    <string>BUSCAR CLIENTES</string>
   </property>
  </action>
  <action name="actionUsuarios">
   <property name="icon">
    <iconset>
     <normaloff>../Escritorio/Iconos/profile_avatar_generic.png</normaloff>../Escritorio/Iconos/profile_avatar_generic.png</iconset>
   </property>
   <property name="text">
    <string>Usuarios</string>
   </property>
  </action>
  <action name="actionUsuarios_2">
   <property name="icon">
    <iconset>
     <normaloff>../Escritorio/Iconos/profile_avatar_generic.png</normaloff>../Escritorio/Iconos/profile_avatar_generic.png</iconset>
   </property>
   <property name="text">
    <string>Usuarios</string>
   </property>
  </action>
  <action name="actionClientes">
   <property name="icon">
    <iconset>
     <normaloff>../Escritorio/Iconos/profile_loading.png</normaloff>../Escritorio/Iconos/profile_loading.png</iconset>
   </property>
   <property name="text">
    <string>Clientes</string>
   </property>
  </action>
  <action name="actionCobradores">
   <property name="icon">
    <iconset>
     <normaloff>../Escritorio/Iconos/profile_avatar_ninja.png</normaloff>../Escritorio/Iconos/profile_avatar_ninja.png</iconset>
   </property>
   <property name="text">
    <string>Cobradores</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
